src/modules/infobox/components/ImportContent.vue

```diff
@@ -0,0 +1,308 @@
+<template>
+    <div data-infobox="height-reference" class="import-overlay-content">
+        <div class="input-group d-flex">
+            <input
+                ref="importInput"
+                type="text"
+                class="form-control import-input rounded-end"
```

Comment:
Just a bit picky, but in case we have the `clear` button, would be nice to not have the `rounded-end` on the input:
![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/cb375bfb-7a1e-43a4-ab19-2589cae3dc99)
It looks in this case a bit uggly.

Reply:
fixed